gherkin/java/src/main/java/io/cucumber/gherkin/GherkinDialectProvider.java

```diff
@@ -44,18 +24,16 @@ public GherkinDialect getDefaultDialect() {
 
     public GherkinDialect getDialect(String language, Location location) {
         requireNonNull(language);
+        GherkinDialect dialect = GherkinDialects.DIALECTS.get(language);
 
-        JsonValue languageObject = DIALECTS.get(language);
-        if (languageObject == null) {
+        if (dialect == null) {
             throw new ParserException.NoSuchLanguageException(language, location);
```

Comment:
Would be nice if we could throw this parser exception elsewhere and have this method return `Optional<GherkinDialect>`.

Reply:
Ok